seed/models/analysis_input_files.py

```diff
@@ -22,6 +29,6 @@ class AnalysisInputFile(models.Model):
         (BUILDINGSYNC, 'BuildingSync'),
     )
 
-    file = models.FileField(upload_to="analysis_input_files", max_length=500)
+    file = models.FileField(upload_to=analysis_input_path, max_length=500)
```

Comment:
I thought it might be nice to break out input files into subdirectories according to the analysis id, rather than having all input files for all analyses living in the same place. Not sure if this is necessary/that important, did it without thinking too much about it, welcoming feedback

Reply:
I like the idea of analysis subdirectories by id, the `uploads` directory is currently a structureless mess with 2951 files all in the same directory and lots of "unique" strings appended to prevent overwriting older files.  The only uploads subdirectory is `pm_imports` with another structureless directory of 1074 files, that makes it a pain to find the actual file you're looking for when you need it.

I'm definitely in favor of a comment that warns against changes to that function that could break existing paths